charts/tensorleap/charts/engine/templates/streaming-handler-deployment.yaml

```diff
@@ -25,6 +25,11 @@ spec:
     spec:
       containers:
         - image: {{ .Values.image }}:{{ .Values.image_tag }}
+          resources:
+            requests:
+              memory: 500Mi
+            limits:
+              memory: 1Gi
```

Comment:
we should be able to add these params to settings such as engine job template override